api_client/python/tools/timesketch_importer.py

```diff
@@ -0,0 +1,269 @@
+# Copyright 2020 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A simple frontend to the Timesketch data importer.
+
+This tool is limited to upload files to Timesketch, which is only
+part of the capabilities of the importer API.
+"""
+from __future__ import unicode_literals
+
+import argparse
+import getpass
+import logging
+import os
+import sys
+
+from typing import Dict
+
+import yaml
+
+from timesketch_api_client import client
+from timesketch_api_client import importer
+from timesketch_api_client import sketch
+
+
+logging.basicConfig(level=os.environ.get('LOGLEVEL', 'INFO'))
+
+
+def get_client(
```

Comment:
I would name it get_api_client, but up to you

Reply:
done